contentcuration/contentcuration/frontend/shared/views/LicenseDropdown.vue

```diff
@@ -15,10 +15,10 @@
         :readonly="readonly"
         :rules="licenseRules"
         :placeholder="placeholder"
-        menu-props="offsetY"
+        :menu-props="{ offsetY: true, lazy: true, top: true, zIndex: 4 }"
```

Comment:
Is the reason that `top: true` is set here because it is near the bottom of the page? i.e. a space issue?

Reply:
Yes that's correct. I tried `offset-overflow` which should flip it automatically, but it didn't actually do that, so I just made it so this always shows above.